### PR TITLE
[AIRFLOW-2538] Update faq doc on how to reduce airflow scheduler latency

### PR DESCRIPTION
**Pull request by feng-tao**
Make sure you have checked _all_ steps below.

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-2538
    - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a JIRA issue.

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:
Update the faq doc on how to reduce airflow scheduler latency. This comes from our internal production setting which also aligns with Maxime's email(https://lists.apache.org/thread.html/%3CCAHEEp7WFAivyMJZ0N+0Zd1T3nvfyCJRudL3XSRLM4utSigR3dQ@mail.gmail.com%3E).

### Tests
- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits
- [ ] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### Documentation
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.

### Code Quality
- [ ] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by feng-tao** [comment]
PTAL @mistercrunch

File: docs/faq.rst [hunk 1]
```diff
@@ -162,10 +162,18 @@ How can we reduce the airflow UI page load time?
...
 ---------------------------------------------------------------------------------------------
 
 This means ``explicit_defaults_for_timestamp`` is disabled in your mysql server and you need to enable it by:
 
 #. Set ``explicit_defaults_for_timestamp = 1`` under the mysqld section in your my.cnf file.
 #. Restart the Mysql server.
+
+
+How to reduce airflow dag scheduling latency in production?
+------------------------------------------------------------
```
Comment (Fokko) [inline docs/faq.rst]: Could you fix the length of this line? Otherwise it will throw an error when compiling the docs.

File: docs/faq.rst [hunk 2]
```diff
@@ -162,10 +162,18 @@ How can we reduce the airflow UI page load time?
...
+
+Scheduler will spawn multiple threads in parallel to schedule dags. This is controlled by ``max_threads`` with default value of 2. User should increase this value to a larger value(e.g same number of cpu where scheduler runs) in production.
```
Comment (Fokko) [inline docs/faq.rst]: Maybe suggest `cores - 1` to leave some headspace for the other processes? This is quite common afaik.

**Review by Fokko — CHANGES_REQUESTED** [review]
Thanks for taking the time to update the docs, I have two small comments.

File: docs/faq.rst (hunk 2 above)
Comment (feng-tao) [reply docs/faq.rst]: Thanks @Fokko for the quick comment. PR updated.

File: docs/faq.rst (hunk 1 above)
Comment (feng-tao) [reply docs/faq.rst]: I use online rst editor which the doc seems to render fine. My understanding is that as long as the length of line is longer than the txt, then we are fine?

Comment (Fokko) [reply docs/faq.rst]: Yes, so it should be:
```
How to reduce airflow dag scheduling latency in production?
-----------------------------------------------------------
```
Would not be my design choice :)

Comment (feng-tao) [reply docs/faq.rst]: Got you :) pr updated.

**Comment by Fokko** [comment]
Nice one, lets wait if @mistercrunch has some suggestions.

Please also limit the commit subject line to 50 chars, and give a more elaborate description in the commit body.

**Comment by feng-tao** [comment]
PTAL @Fokko

**Pull request closed without merging**